tests/alerts_test.py

```diff
@@ -1190,6 +1190,206 @@ def test_slack_uses_list_of_custom_slack_channel():
     assert expected_data2 == json.loads(mock_post_request.call_args_list[1][1]['data'])
 
 
+def test_slack_attach_kibana_discover_url_when_generated():
+    rule = {
+        'name': 'Test Rule',
+        'type': 'any',
+        'slack_attach_kibana_discover_url': True,
+        'slack_webhook_url': 'http://please.dontgohere.slack',
```

Comment:
I see this was used in previous tests. Nvm

~example.com is sort of designed for this usecase. I might suggest using `https://example.com/slack_webhook`~

> ~As described in RFC 2606 and RFC 6761, a number of domains such as example.com and example.org are maintained for documentation purposes. These domains may be used as illustrative examples in documents without prior coordination with us. They are not available for registration or transfer.~